cli_tests/app.sh

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env bats
+
+# To run locally:
+# export KETCH_COMMAND=<location of ketch binary>
+# assure you have a kubernetes cluster running w/ traefik, cert manager, etc. (see ketch getting started docs)
+# assure the ketch cli is compiled (make ketch)
+# assure you have bats installed locally (via apt, brew, etc.)
+# ./cli_tests/app.sh
+
+setup() {
+    if [[ -z "${KETCH_COMMAND}" ]]; then
```

Comment:
I feel like `KETCH_COMMAND` is a little misleading since I thought it was referring to something like app or framework at first. Maybe a variable name like KETCH_EXECUTABLE_PATH or something would be a bit more clear?

Reply:
I like it. Change made.